### PR TITLE
ERM-1176: Improvements to the Amendment preview pane presentation

### PR DESCRIPTION
**Pull request by EthanFreestone**

### CODE REVIEW

File: src/routes/ViewAmendmentRoute.js [hunk 1]
```diff
@@ -106,6 +106,7 @@ class ViewAmendmentsRoute extends React.Component {
 
   urls = {
     editAmendment: this.props.stripes.hasPerm('ui-licenses.licenses.edit') && (() => `/licenses/${this.props.match.params.id}/amendments/${this.props.match.params.amendmentId}/edit${this.props.location.search}`),
+    licenseView: id => `/licenses/${id}`
```
Comment (aditya-matukumalli) [inline src/routes/ViewAmendmentRoute.js:109]: If you want to keep the `urls` object declared here as the source of truth for all the child components of this route, then 
you could import `urls` from utils and assign `licenseView: urls.licenseView`. That way we wont dont repeat the code multiple times.

**Review by aditya-matukumalli — CHANGES_REQUESTED** [review]

File: src/components/Amendment/Amendment.js [hunk 2]
```diff
@@ -146,13 +146,14 @@ export default class Amendment extends React.Component {
     return (
       <Pane
         actionMenu={this.renderActionMenu}
+        appIcon={<AppIcon app="licenses" iconKey="amendment" size="small" />}
```
[the remark this answers is not in the document]
Comment (EthanFreestone) [reply src/components/Amendment/Amendment.js:149]: Ah, this was a mistake, changing now

File: src/routes/ViewAmendmentRoute.js (hunk 1 above)
Comment (EthanFreestone) [reply src/routes/ViewAmendmentRoute.js:109]: I did try this and it failed. I'll retry though

Comment (EthanFreestone) [reply src/routes/ViewAmendmentRoute.js:109]: Per slack, not doing this as both are unique to this route, neater to leave as is

**Pull request merged**